proto/exocore/assets/v1/tx.proto

```diff
@@ -117,7 +117,8 @@ message StakerAllAssetsInfo {
 // OperatorAssetInfo defines the information for a single asset,
 // for an operator.
 message OperatorAssetInfo {
-  // total_amount is the total amount of the asset deposited.
+  // total_amount is the total amount of the asset deposited, which excludes the
+  // wait_unbonding_amount
```

Comment:
Because it represents all delegated amounts of the asset pool, it will be used to calculate the delegated amount of a specified staker from its share. I will add a code comment to clarify this.